tails_files/install.sh

```diff
@@ -20,15 +46,48 @@ apt-get update
 apt-get install -y build-essential
 gcc -o $SCRIPT_BIN securedrop_init.c
 
-# copy init scripts
-cp securedrop_init.py $SCRIPT_PY
-cp .xsessionrc $INSTALL_DIR
-
+# copy icon, launchers and scripts
 cp securedrop_icon.png $INSTALL_DIR
+cp document.desktop $INSTALL_DIR
+cp source.desktop $INSTALL_DIR
+cp .xsessionrc $INSTALL_DIR
+cp securedrop_init.py $SCRIPT_PY
 
-# prepare torrc_additions
-cp torrc_additions $ADDITIONS
-gedit $ADDITIONS > /dev/null 2>&1
+if $ADMIN; then
+  DOCUMENT=`cat $ANSIBLE/app-document-aths | cut -d ' ' -f 2`
+  SOURCE=`cat $ANSIBLE/app-source-ths`
+  APPSSH=`cat $ANSIBLE/app-ssh-aths | cut -d ' ' -f 2`
+  MONSSH=`cat $ANSIBLE/mon-ssh-aths | cut -d ' ' -f 2`
+  cat $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths >> $ADDITIONS
```

Comment:
Note that if this script were run twice, you could end up appending these lines to the `$INSTALL_DIR/torrc_additions` *twice*. If you do that, when Tor restarts it will crash silently due to a particularly nasty bug in Tor (ask me how I know this). An easy solution would be to create the file with `>` instead of appending to it with `>>`.